open_schools_platform/common/services.py

```diff
@@ -58,10 +58,34 @@ class BaseQueryHandler:
     Contain basic query checks
     """
```

Comment:
please, extend docstring)
don't forget about rules, this class works only if rules has predicate_input_type_check